_posts/en/newsletters/2020-06-10-newsletter.md

```diff
@@ -0,0 +1,329 @@
+---
+title: 'Bitcoin Optech Newsletter #101'
+permalink: /en/newsletters/2020/06/10/
+name: 2020-06-10-newsletter
+slug: 2020-06-10-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter links to a blog post and paper about using
+eclipse attacks against Bitcoin software in order to steal from LN
+channels and describes the history and impact of a fee overpayment
+attack that affects hardware wallets.  Also included are our regular
+sections with the summary of a Bitcoin Core PR Review Club meeting,
+descriptions of recent software releases, and summaries of recent
+changes to popular Bitcoin infrastructure projects.
+
+## Action items
+
+- **Check hardware wallet compatibility:** if your software or your
+  processes allow using Trezor devices to spend segwit inputs, check
+  whether your system remains compatible with [Trezor's latest firmware
+  update][trezor update].  Other hardware wallets may release similar
+  updates in the future; contact the manufacturers for more information
+  about their plans.  See the *news* item below about the fee
+  overpayment attack for the motivation for this
+  change.
+
+## News
+
+- **Time dilation attacks against LN:** Gleb Naumenko and Antoine Riard
+  [posted][naumenko td] to the Bitcoin-Dev mailing list a summary of a
+  [blog post][td blog] they wrote that itself summarizes a [paper][td
+  paper] they authored about using [eclipse attacks][topic eclipse attacks]
+  to steal money from LN channels.  This extends the analysis by Riard
+  described in [Newsletter #77][news77 eclipse].  In short, an attacker
+  gains control over all of an LN node's connections to the Bitcoin P2P
+  network and delays the relay of new block announcements to the victim.
+  After the victim's view of the block chain is far enough behind the
+  public consensus view, the attacker closes their channels with the
+  victim in an outdated state that pays the attacker more than the
+  latest state would.  The eclipse attack is used to prevent the victim
+  from seeing the close transactions until after the dispute period is
+  over and the attacker has withdrawn their illicit gains to an address
+  entirely under the attacker's control.
+
+    The paper describes an attack that can steal from lightweight
+    clients in as little as two hours.  For attacks against LN nodes
+    backed by a local Bitcoin full verification node, their shortest
+    duration attack requires eclipsing the victim for at least 24 hours.
```

Comment:
In fact that's few hours, depending on implementation time-locks (see Table 1 : Default timelock (in blocks), timeout policy). 

You can also note, due to the lack of hardened eclipse protection in lightweight client and low-number of deployed lightweight servers, this attack is practical with high-chances of success _today_, assuming you have a channel with the victim.